builds/cmake/Modules/Findwildmidi.cmake

```diff
@@ -0,0 +1,10 @@
+find_path(WILDMIDI_INCLUDE_DIR wildmidi_lib.h)
+find_library(WILDMIDI_LIBRARY NAMES wildmidi libwildmidi WildMidi libWildMidi)
+
+include(FindPackageHandleStandardArgs)
```

Comment:
wildmidi has currently no pkg-config support (but i want to fix that upstream so letting it stay :D)